hermes/bin/src/runtime_context.rs

```diff
@@ -16,18 +16,24 @@ pub(crate) struct HermesRuntimeContext {
 
     /// module's execution counter
     exc_counter: u32,
+
+    /// App Virtual file system
+    #[allow(dead_code)]
+    vfs: Option<Vfs>,
```

Comment:
I think it shouldn't be optional.
I know that it complicates initialisation of the runtime context for integration tests, but we still need an even even there.